### PR TITLE
Major U.S. credit unions

### PR DESCRIPTION
**Pull request by 1ec5**
Added U.S. credit unions with 50 or more branches. (A bunch more credit unions just barely miss that threshold.)

### CODE REVIEW

File: brands/amenity/bank.json [hunk 1]
```diff
@@ -4133,6 +4165,18 @@
       "name": "Pekao SA"
     }
   },
+  "amenity/bank|PenFed Credit Union": {
+    "countryCodes": ["us"],
+    "matchNames": ["penfed"],
```
Comment (matkoniecz) [inline brands/amenity/bank.json]: sounds like `short_name` to me.

File: brands/amenity/bank.json [hunk 2]
```diff
@@ -4530,6 +4575,28 @@
...
+      "amenity": "bank",
+      "brand": "Security Service Federal Credit Union",
+      "brand:wikidata": "Q7444993",
+      "brand:wikipedia": "en:Security Service Federal Credit Union",
+      "name": "Security Service Federal Credit Union",
+      "short_name": "SSFCU"
+    }
+  },
+  "amenity/bank|Service Credit Union": {
+    "countryCodes": ["de", "us"],
```
Comment (tas50) [inline brands/amenity/bank.json]: Are you sure this is in both europe and the US?

Comment (mds08011) [reply brands/amenity/bank.json]: Surprising but per [Wikipedia](https://en.wikipedia.org/wiki/Service_Credit_Union) (emphasis mine):
>As of 2018, Service Credit Union has 50 locations (35 in New Hampshire, 1 in Massachusetts, 1 in North Dakota **and 13 on US military installations in Germany**)

Comment (tas50) [reply brands/amenity/bank.json]: That actually makes perfect sense. I'm sure USAA probably has a few branches in Germany for the same reason.

**Comment by bhousel** [comment]
Looks great, thanks!

**Pull request merged**